packages/popover/src/vaadin-popover-overlay-mixin.js

```diff
@@ -68,15 +35,6 @@ export const PopoverOverlayMixin = (superClass) =>
         return;
       }
 
-      // Copy custom properties from the owner
```

Comment:
This is no longer needed as offset custom CSS properties will inherit to the overlay via cascade.